Examples/MAX78000/CNN/kws20_demo/main.c

```diff
@@ -61,13 +61,17 @@
 #include "cnn.h"
 #ifdef BOARD_FTHR_REVA
 #include "tft_ili9341.h"
+#ifdef ENABLE_CODEC_MIC
+#include "i2c.h"
+#include "max9867.h"
```

Comment:
@aniktash @vicloginov FYI Jesse added some codec drivers to [`Libraries/MiscDrivers/CODEC`](https://github.com/Analog-Devices-MSDK/msdk/tree/main/Libraries/MiscDrivers/CODEC) in #240.  We should use those drivers instead

Reply:
updated